example_scenes/manim.cfg

```diff
@@ -0,0 +1,12 @@
+[log.color]
```

Comment:
I don't know how I feel about having a dot inside the name of a section. This is purely a matter of opinion, but I think this section should just be called `logger` or `rich`.